pandas/tests/test_multilevel.py

```diff
@@ -2381,6 +2381,39 @@ def test_iloc_mi(self):
         tm.assert_frame_equal(result, expected)
 
 
+@pytest.fixture
+def sparse_df():
```

Comment:
Any particular sparse reshaping tests you had in mind? There's one `test_stack_sparse_frame` in `sparse.test_frame`.